### PR TITLE
docs: Improve Kotlin documentation

### PR DESCRIPTION
**Pull request by jan-tennert**
## I have read the [CONTRIBUTING.md](https://github.com/supabase/supabase/blob/master/CONTRIBUTING.md) file.

YES

## What kind of change does this PR introduce?

Docs update

## What is the new behavior?

Small tweaks/changes to the Kotlin documentation

## Additional context

I discovered some weird stuff in the Kotlin docs, but I didn't know where to fix that:
Blackslashes in code blocks in the initializing section:
grafik
grafik
Lowercase Kotlin name in the title:
grafik

### CODE REVIEW

File: apps/docs/spec/supabase_kt_v2.yml [hunk 1]
```diff
@@ -628,12 +643,14 @@ functions:
...
     $ref: '@supabase/postgrest-js."lib/PostgrestQueryBuilder".PostgrestQueryBuilder.delete'
     notes: |
+      Perform a DELETE on the table or view.
       - `delete()` should always be combined with a [filter](/docs/reference/kotlin/using-filters) block to target the item(s) you wish to delete.
       - If you use `delete()` with filters and you have
         [RLS](/docs/learn/auth-deep-dive/auth-row-level-security) enabled, only
         rows visible through `SELECT` policies are deleted. Note that by default
         no rows are visible, so you need at least one `SELECT`/`ALL` policy that
         makes the rows visible.
+      - By default, `upsert` will not return the inserted data. If you want to return the inserted data, you can use the `select()` method inside the request.
```
Comment (J0) [inline apps/docs/spec/supabase_kt_v2.yml:653]: Confirming that this is relevant as it seems slightly out of place

**Review by J0 — APPROVED** [review]
Thanks!

File: apps/docs/spec/supabase_kt_v2.yml (hunk 1 above)
Comment (jan-tennert) [reply apps/docs/spec/supabase_kt_v2.yml]: Yea, that shouldn't be about upserting/inserting. Fixed it.

**Comment by charislam** [comment]
Thanks @jan-tennert! I fixed the braces problem in a [separate PR](https://github.com/supabase/supabase/pull/29063) so let's get this one in! 🚀 🚢

**Pull request merged**